c/serializer/samples/remote_monitoring/windowsce/remote_monitoring.sln

```diff
@@ -0,0 +1,22 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 2013
```

Comment:
If you have VS 2015 it would be nice to upgrade these, otherwise we can also do it after the merge.